graviti/openapi/tag.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Interfaces about the tag."""
+
+from typing import Any, Dict
+from urllib.parse import urljoin
+
+from graviti.openapi.request import URL_PATH_PREFIX, open_api_do
+
+
+def create_tag(url: str, access_key: str, dataset_name: str, name: str, revision: str) -> None:
+    """Execute the OpenAPI `POST /v1/datasets/{dataset_name}/tags`.
+
+    Arguments:
+        url: The URL of the graviti website.
+        access_key: User's access key.
+        dataset_name: Name of the dataset, unique for a user.
+        name: The tag name to be created for the specific commit.
+        revision: The information to locate the specific commit, which can be the commit id,
+            the branch name, or the tag name.
+
+    Examples:
+        >>> create_tag(
+        ...     "https://graviti.com/",
```

Comment:
we update the url to `https://api.graviti.com/`

Reply:
please update it for all your PR

Reply:
some PR has been landed, should we modify them including URL_PRE_FIX, examples in another PR

Reply:
ok